server/worker/macos_setup_assistant.go

```diff
@@ -101,7 +101,7 @@ func (m *MacosSetupAssistant) runProfileChanged(ctx context.Context, args macosS
 	// re-generate and register the profile with Apple. Since the profile has been
 	// updated, then its profile UUID will have been cleared, so this single call
 	// will do both tasks.
-	profUUID, _, err := m.DEPService.EnsureCustomSetupAssistantIfExists(ctx, team)
+	profUUID, _, err := m.DEPService.EnsureCustomSetupAssistantIfExists(ctx, team, "TODO ABM TOKEN ORG NAME")
```

Comment:
@mna which org name would be passed here? Would we loop over the orgs associated with each host in
the team, or use the org for which this team is a default team?

Reply:
Good question! I think the work I'll be doing this PM on the worker setup assistant job will help identify that.